src/firmware/app/control/trajectory_planner.c

```diff
@@ -178,12 +52,21 @@ app_trajectory_planner_interpolateConstantPeriodPositionTrajectory(
             const float delta_y =
```

Comment:
This linear interpolation is now quite large (in terms of LOC). Could we pull it out into a function?

Reply:
Moved into it's own math function 👍